### DISCUSSION TITLE
Laurenii

### DISCUSSION THREAD

**Discussion by StroescuDiana** [peviitor-ro/community#54] [Scrapers Teams]
1. Adriana Cionca
2. Alexandra Nicoara
3. Dana Catavei
4. Emese 
5. Adi Vanca
6. Ana Maria Lazar
7. Adina Boeru

**Comment by Mede375** [comment]
Avem nevoie de un nume de echipa! Numele echipei va fi si numele repo

**Comment by sebiboga** [comment]
Laurenii sau Laurienii ?
